### PR TITLE
#89 Implementing bike order feature

### PR DESCRIPTION
**Pull request by Mnx458**

### CODE REVIEW

File: bike-erp/backend/src/main/services/orderService/BikeOrderService.ts [hunk 1]
```diff
@@ -34,10 +35,15 @@ export class BikeOrderService {
...
                 }
-                    BikeOrderService.getBikeDao().createBike(bike.price, bike.size, bike.color, bike.description, bike.grade, bike.quantity)
-                    .catch(error => {
-                        return rejects(error.message)
-                    })
+                //Posting all the bikes created with their components Id.
+                try {
+                    const response = await BikeOrderService.getBikeDao().createBike(bike.price, bike.size, bike.color, bike.description, bike.grade, bike.quantity);
+                    await BikeOrderService.getBikeDao().linkBikeToComponents(response.bikeId, bike.handle_id, bike.wheel_id, bike.frame_id, bike.seat_id, bike.drive_train_id);
+                    bikeIdList.push(response.bikeId);
```
Comment (nasaku898) [inline bike-erp/backend/src/main/services/orderService/BikeOrderService.ts:42]: Don't forget to integrate the account_receivable into the order bike process

Comment (Mnx458) [reply bike-erp/backend/src/main/services/orderService/BikeOrderService.ts:42]: yes, will do that on another branch

**Review by nasaku898 — CHANGES_REQUESTED** [review]
Overall looks good, but could you add semi-colon in your frontend files for the consistency purposes and coding convention.

File: bike-erp/src/redux/types/OrderBikeTypes/orderBikeTypes.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export const ADD_BIKE: string = "ADD_BIKE"
```
Comment (nasaku898) [inline bike-erp/src/redux/types/OrderBikeTypes/orderBikeTypes.ts:1]: Add semi-column here

**Review by nasaku898 — APPROVED** [review]
Great work!

**Review by Manraj45 — APPROVED** [review]
Good Job!

**Pull request merged**